apps/bestofjs-nextjs/src/lib/global.d.ts

```diff
@@ -104,13 +104,19 @@ declare namespace BestOfJS {
     projects: BestOfJS.Project[];
   }
 
-  type HallOfFameMember = {
-    username: string;
+  interface RawHallOfFameMember {
     avatar: string;
-    followers: numbers;
     bio: string;
     blog: string;
-  };
+    followers: numbers;
+    name: string;
+    projects: string[];
+    username: string;
+  }
+
+  interface HallOfFameMember extends RawHallOfFameMember {
```

Comment:
and this is the App's data model?

Reply:
And yes that is the populated object, the Hall of Game member with the related projects, ready to be consumed by the frontend application.